### ISSUE TITLE
Correct licensing info in OSM wiki (better display)

### ISSUE THREAD

**Issue by rugk** [rugk/streetcomplete-quest-svgs#12]
Most uploads do just have the text template stating the license, see: https://wiki.openstreetmap.org/wiki/File:StreetComplete_quest_max_height.svg

Better would be something like this:
```
== Licensing ==
{{CC-BY-SA-4.0}}
Created by [[User:Westnordost|Tobias Zwick]]<br>
Source: https://github.com/westnordost/StreetComplete/blob/master/res/quest_icons.svg
```

as shown in https://wiki.openstreetmap.org/wiki/File:StreetComplete_quest_street_surface.png

**Comment by Binnette** [comment]
Seems good to me

**Comment by rugk** [comment]
Yeah, I guess I've created this issue here to track that we have not converted all OSM wiki pages to that format already. The doc here (https://github.com/rugk/streetcomplete-quest-svgs/commit/59ede3720e868e0900e64c55e91d8c9977533832) is already adjusted, in any case.
